doc/api.rst

```diff
@@ -56,6 +56,7 @@ Utils
    get_entity_vals
    get_kinds
    BIDSPath
+   delete_scan
```

Comment:
I don't think this should be in utils, it should be above somewhere in the same place you have write and read functions